stdlib/float.mli

```diff
@@ -608,6 +621,20 @@ module Array : sig
   (** [map_from_array f a] applies function [f] to all the elements of [a],
       and builds a floatarray with the results returned by [f]. *)
 
+  val to_array : t -> float array
+  (** [to_array a] builds the float array equivalent of a:
+      [[a.(0); a.(1); ...; a.(length a - 1) |]]. *)
```

Comment:
Maybe simpler would be to say: `[to_array a] is [[|a.(0); ...; a.(length a - 1)|]].` (also, missing vertical bar on the left).

Reply:
Indeed, then this is also a problem in the documentation of `map_to_array`.